README.md

```diff
@@ -152,6 +152,13 @@ $ mkdir docker-compose_datadir
 $ alias dcd="docker-compose -f docker-compose.yml -f docker-compose-shared_datadir.yml"
 $ dcd up -d
 ```
+Note: In case you want to start the docker composition and being able to test some layers, 
+you can create a symbolic link to the data/release datadir that is part of the geoserver_submodule
+instead of creating the docker-compose_datadir emtpy folder.
+
+```bash
```

Comment:
this looks like a bad idea, you'll end up with a lot of changed files in the submodule. I'd rather:
```
cp -rf ./geoserver_submodule/geoserver/data/release /tmp/datadir
ln -s /tmp/datadir docker-compose_datadir
```